ui/store/actions.js

```diff
@@ -671,6 +671,23 @@ const updateMetamaskStateFromBackground = () => {
   });
 };
 
+export function updatePreviousGasParams(txId, previousGasParams) {
```

Comment:
This should be updated to reflect the changes I made in the other method, and return the updated transaction